spring-cloud-vault-core/src/main/java/org/springframework/cloud/vault/VaultResponse.java

```diff
@@ -16,11 +16,11 @@
 
 package org.springframework.cloud.vault;
 
-import java.util.Map;
+import lombok.Data;
```

Comment:
Absolutely none.  I imported the eclipse style sheet specified and this happened.  Will revert.  Any configuration I could import to align with your standards?